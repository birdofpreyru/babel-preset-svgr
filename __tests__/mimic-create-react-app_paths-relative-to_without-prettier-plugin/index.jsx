/**
 * Tests the preset used without any options.
 */

import React from 'react';
import ReactDOM from 'react-dom/server';

import logoPath, { ReactComponent } from './logo.svg';

const EXPECTED_LOGO_PATH = 'logo.svg';

test('SVG import works as expected', () => {
  expect(logoPath).toBe(EXPECTED_LOGO_PATH);
  const code = ReactDOM.renderToString(<ReactComponent />);
  expect(code).toMatchSnapshot();
});
