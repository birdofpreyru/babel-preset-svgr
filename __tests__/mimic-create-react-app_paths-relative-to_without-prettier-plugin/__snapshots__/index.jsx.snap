// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SVG import works as expected 1`] = `"<svg width=\\"150.517\\" height=\\"188.611\\" viewBox=\\"0 0 39.824 49.903\\" data-reactroot=\\"\\"><path fill=\\"#f5f5f5\\" d=\\"M0 0h39.824v49.903H0z\\"></path><g aria-label=\\"P\\" style=\\"line-height:120.00000477%;-inkscape-font-specification:Oswald\\" font-weight=\\"400\\" font-size=\\"99.751\\" font-family=\\"Oswald\\" letter-spacing=\\"0\\" word-spacing=\\"0\\" fill=\\"#313131\\" stroke-width=\\"0.399\\"><path d=\\"M14.119 34.474h5.893c4.281 0 7.547-1.214 9.798-3.642 2.251-2.428 3.377-5.826 3.377-10.196 0-4.855-1.06-8.54-3.178-11.057-2.075-2.516-5.43-3.774-10.064-3.774H6.637l-.035 34.02H0V0h39.824v39.824H14.112z\\"></path><path d=\\"M17.562 29.11c2.207 0 3.906-.242 5.098-.728 1.236-.485 2.119-1.324 2.648-2.516.53-1.191.795-2.913.795-5.164 0-2.736-.199-4.745-.596-6.025-.397-1.28-1.192-2.185-2.384-2.714-1.147-.53-2.98-.795-5.495-.795h-3.509v17.943z\\"></path></g><text style=\\"line-height:120.00000477%;-inkscape-font-specification:Oswald\\" x=\\"43.138\\" y=\\"136.474\\" font-weight=\\"400\\" font-size=\\"6.063\\" font-family=\\"Oswald\\" letter-spacing=\\"0\\" word-spacing=\\"0\\" fill=\\"#313131\\" stroke-width=\\"0.265\\" transform=\\"translate(-43.52 -87.656)\\"><tspan x=\\"43.138\\" y=\\"136.474\\">Dr. Pogodin Studio</tspan></text></svg>"`;
